src/monitoring_service/handlers.py

```diff
@@ -310,7 +310,17 @@ def monitor_new_balance_proof_event_handler(event: Event, context: Context) -> N
 
 def monitor_reward_claim_event_handler(event: Event, context: Context) -> None:
     assert isinstance(event, ReceiveMonitoringRewardClaimedEvent)
-    log.info("Received MSC RewardClaimed event", evt=event)
+
+    if event.ms_address == context.ms_state.address:
+        log.info(
+            "Successfully claimed reward", amount=event.amount, reward_id=event.reward_identifier
+        )
+    else:
```

Comment:
someone told me that we should not use `else` in the Raiden code but smarter `ifs`

Reply:
But in this case it would then log both messages which is not what I want